Pod/Classes/Configuration.swift

```diff
@@ -1,56 +1,48 @@
 //
 //  Configuration.swift
-//  Pods
+//  TouchVisualizer
 //
 //  Created by MORITA NAOKI on 2015/04/11.
-//
+//  Copyright (c) 2015年 molabo. All rights reserved.
 //
 
 import UIKit
 
 public struct Configuration {
+    private struct Constants {
+        static let defaultColor = UIColor(red: 52.0/255.0, green: 152.0/255.0, blue: 219.0/255.0, alpha: 0.8)
+    }
     
-    static let defaultColor = UIColor(red: 52/255.0, green: 152/255.0, blue: 219/255.0, alpha: 0.8)
-    
-    /**
-    Color of touch points.
-    */
-    public var color: UIColor? = defaultColor
+    // Color of touch points
```

Comment:
I want to use `reStructured` comment. It is swift official. My comment is not enough but I want to improve it step by step.

http://nshipster.com/swift-documentation/